app/helpers/api_keys_helper.rb

```diff
@@ -1,8 +1,8 @@
 module ApiKeysHelper
-  def mask_api_key(api_key)
-    return "****" if api_key.api_key_id.nil? || api_key.api_key_id.length < 5
+  def mask_api_key(api_key_id)
```

Comment:
I prefer passing the whole object as it is consistent with the other helper methods, and avoids the views having to know about the structure of the object so it can pass the correct value to this method.

Reply:
Fixed :tada: